gatsby/content/blog/2022/04/2022-04-05-synapse-1.56.mdx

```diff
@@ -0,0 +1,82 @@
+---
+date: '2022-04-05'
+title: Synapse 1.56 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+Happy release day everyone! [Synapse
+1.56](https://github.com/matrix-org/synapse/releases/tag/v1.56.0) is out! Let's
+have a look at what's new in this release.
+
+## Additional restrictions around registration
+
+One common scenario of abuse we have seen on Matrix over the years is attackers
+making use of public homeservers with open registration to automatically create
+multiple accounts in order to evade sanctions. Sometimes, this can lead to these
+public homeservers being banned across multiple public rooms despite their
+administrator(s) not being directly at fault.
+
+In order to mitigate this, starting from this release, Synapse will refuse to
+start if it is configured with open registration with no additional
+verification. This means users wishing to register on these homeservers will
+need to authenticate themselves, either via
+[email](https://github.com/matrix-org/synapse/blob/8a519f8abc6de772167c2cca101d22ee2052fafc/docs/sample_config.yaml#L1285),
+[recaptcha](https://matrix-org.github.io/synapse/v1.56/CAPTCHA_SETUP.html) or
+[registration
+tokens](https://matrix-org.github.io/synapse/v1.56/usage/administration/admin_api/registration_tokens.html).
```

Comment:
Unsure, but maybe note you can opt out of this requirement. There may still be valid uses for open registration in situations where you understand what you're doing; we don't necessarily want to make it sound like we're deliberately taking away flexibility, just that we're trying our best to make it hard to leave your server as an open relay unconsciously.

Reply:
Sure, but I don't think this is something we want to recommend to most people. I assume if someone starts worrying about that they'll go read the upgrade notes for all the info and will see it can be disabled.

Reply:
Maybe 'by default' is the phrasing we could use here. I'm just wary of giving the image that 'the synapse developers know better than you and won't let you do what you want'

Reply:
Good point, have changed into "will refuse to start by default if..."